src/implementations/twig/components/datepicker/datepicker.html.twig

```diff
@@ -18,12 +18,12 @@
     - "icons_path" (string) (default: '')
     - "placeholder" (string) (default: 'DD-MM-YYYY')
     - "extra_classes" (string) (default: '')
-    - "extra_attributes" (array) (default: []): format: [
-        {
-          "name" (string) (default: ''),
-          "value" (optional) (string)
-        ...
-      ],
+    - "extra_attributes" (array) (default: []) Extra attributes for the datepicker wrapper
+      - "name" (string) Attribute name, eg. 'data-test'
+      - "value" (optional) (string) Attribute value, eg: 'data-test-1'
+    - "input_extra_attributes" (array) (default: []) Extra attributes for the datepicker input
```

Comment:
Do we need this additional parameter? I mean, what are the extra_attributes for otherwise? For the wrapping div i think we don't really need them and in the end we do expect these attributes to always come as input.extra_attributes, i wouldn't introduce this inconsistency, it feels really unneeded.